content/info/contacts.md

```diff
@@ -3,24 +3,22 @@ title = "Contact Alzymologist Oy"
 date = 2022-07-30
 +++
 
-## Our contacts
+## Lab contacts
 
-The best way to learn more about us is to write to [**contact@zymologia.fi**](mailto:contact@zymologia.fi) <br>
+The best way to learn more about laboratory is to write to [**contact@zymologia.fi**](mailto:contact@zymologia.fi) <br>
```

Comment:
revert this, let them write on all matters

Reply:
Reverted.